cpp/tests/structure/mg_select_random_vertices_test.cpp

```diff
@@ -242,8 +259,8 @@ INSTANTIATE_TEST_SUITE_P(
                           factor (to avoid running same benchmarks more than once) */
   Tests_MGSelectRandomVertices_Rmat,
   ::testing::Combine(
-    ::testing::Values(SelectRandomVertices_Usecase{500, false},
-                      SelectRandomVertices_Usecase{500, false}),
+    ::testing::Values(SelectRandomVertices_Usecase{500, true},
+                      SelectRandomVertices_Usecase{500, true}),
```

Comment:
Why did you set check_correctness for benchmark test?

Reply:
It's already changed in next commit, but not pushed yet.